### DISCUSSION TITLE
Student pack benifits not received after verification

### DISCUSSION THREAD

**Discussion by AlexLin625** [community/community#146435] [GitHub Education]
### Hi everyone,

Hello,

I recently passed the GitHub student verification process on December 2nd. However, before that, my application was rejected several times due to incomplete personal information.

After successfully completing the verification, I noticed that I have not yet received the Student Pack benefits. I am reaching out to the community for assistance in resolving this issue.

Any help would be greatly appreciated!

**Comment by davevad93** [comment]
Hi @AlexLin625 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AlexLin625 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
